_includes/head.html

```diff
@@ -1,9 +1,15 @@
 <title>{% if page.title %} {{ page.title }} - {% endif %}Propel, The Blazing Fast Open-Source PHP 5.5 ORM</title>
 <meta name="language" content="en" >
 <meta http-equiv="content-type" content="text/html; charset=utf-8" >
-<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" >
+<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" > 
 <meta content='width=device-width' name='viewport'>
 
+
+<!-- OpenGraph meta tags -->
+<meta property="og:title" content="Propel, The Blazing Fast Open-Source PHP 5.5 ORM" />
```

Comment:
5.5?
Could we do it without the number?

Reply:
Done :)
cf: #423